websites/V/VtuberWiki/presence.ts

```diff
@@ -0,0 +1,118 @@
+type StaticPagesType = Record<string, PresenceData>;
+
+const presence = new Presence({
+		clientId: "1164189020922843257",
+		appMode: true,
```

Comment:
[`appMode`](https://docs.premid.app/en/dev/presence/class#appmode-deprecated-since-224) is deprecated.
```suggestion
```